comfy_cli/command/nodes.py

```
@@ -0,0 +1,868 @@
+
+app = typer.Typer(no_args_is_help=True, help="Introspect ComfyUI node classes (inputs, outputs, categories).")
+
+
+# ---------------------------------------------------------------------------
+# graph resolution — shared across ls/show/search
+# ---------------------------------------------------------------------------
+
+
+def _resolved_where(where: str | None) -> str:
+    """Apply the full precedence chain: per-command flag > env > config > default."""
+    from comfy_cli import where as where_module
```

Pylint (4.0.5)

[convention] Import outside toplevel (comfy_cli.where)

(C0415)